cmd/kubeadm/app/discovery/file/file.go

```diff
@@ -98,7 +98,7 @@ func ValidateConfigInfo(config *clientcmdapi.Config, discoveryTimeout time.Durat
 			clusterinfoCM, lastError = client.CoreV1().ConfigMaps(metav1.NamespacePublic).Get(context.TODO(), bootstrapapi.ConfigMapClusterInfo, metav1.GetOptions{})
 			if lastError != nil {
 				if apierrors.IsForbidden(lastError) {
-					// If the request is unauthorized, the cluster admin has not granted access to the cluster info configmap for unauthenticated users
+					// If the request is forbidden, the cluster admin has not granted access to the cluster info configmap for current user
```

Comment:
```suggestion
					// If the request fails with a forbidden error, the cluster admin has not granted access to the cluster info configmap for unauthenticated users.
```

- it implies anonymous users not being able to access it. not only the current user.
forbidden.
- the request itself is neither forbidden or unauthorized, it the status code. the request itself just failed.
-

Reply:
@neolit123  OK, got it. How about changing `unauthenticated users` to `unauthorized users`? `unauthenticated users` only include anonymous users, but `unauthorized users` include both anonymous and authenticated users.

Reply:
best to change it to 'anonymous users'. k8s has the concept of 'anonymous auth', and this cm and uses case are exactly about that.

Reply:
@neolit123 ok. Thanks for your review.